### ISSUE TITLE
remove paper-badge

### ISSUE THREAD

**Issue by Zecat** [Zecat/paper-stepper#22]
> `<paper-badge>` is a circular text badge that is displayed on the top right corner of an element

It is used the wrong way, doesn't increase accessibility because the `for`attribute isn't specified, complexifies the element and force to use `this.updateStyles('--paper-badge-background');`.  It should be replaced with simple css.
